assets/files.wxs

```diff
@@ -1958,6 +1958,14 @@
             <File Id="fil6B026F6D5210498B99E5F14386F4F90B" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\Microsoft.PowerShell.ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Validate Remove-LocalUser cmdlet.Errors on remove by invalid SID
```powershell
Expected strings to be the same, but they were different.
Expected length: 65
Actual length:   66
Strings differ at index 0.
Expected: 'UserNotFou...'
But was:  'GroupNotFo...'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalUser.Tests.ps1: line 33
33:         $_.FullyQualifiedErrorId | Should -Be $expectedFqeid
```